src/Menu/SelectMenu.jsx

```diff
@@ -9,10 +9,15 @@ import { ExpandMore } from '../../icons';
 
 export const SELECT_MENU_DEFAULT_MESSAGE = 'Select...';
 
+// this variable is used to focus the menu open button after any menu option is clicked.
+// triggerTarget.current.focus() inside the onCLick() function didn't guarantee element focus.
+let focusMenu = false;
```

Comment:
[confirmation] If there are multiple `SelectMenu` components rendered on a page at once, will having this `focusMenu` variable defined outside of the component itself be an issue, i.e., will changing its value while interacting with one `SelectMenu` impact the behavior of another `SelectMenu` since it's a variable defined outside of the component?

Would moving `focusMenu` inside of the component with a `useRef` value to avoid the above issue make sense (a `useRef` to prevent any re-rendering)?

```jsx
const focusMenuRef = useRef(false);
// ...
focusMenuRef.current = true/false;
```

Reply:
Actually yes, for multiple instances they can have unexpected behavior. So I add ref instead of variable